src/main/java/seedu/duke/component/VoltageSource.java

```diff
@@ -7,4 +7,14 @@ public VoltageSource(double value) {
         super(value);
         siUnit = SI_UNIT;
     }
+
+    /**
+     * Sets the value of the voltage source.
+     *
+     * @param value Value of Component.
+     */
+    @Override
+    public void setValue(double value) {
+        super.setValue(value);
+    }
```

Comment:
Because of inheritance this is not needed.
```suggestion
```

Reply:
Yup noted. I didn't see that before.